integration/setup/setup.go

```diff
@@ -69,5 +72,19 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
+	// Ensure Chart CRD creation
+	{
+		c := k8scrdclient.Config{
+			K8sExtClient: config.Host.ExtClient(),
+			Logger:       config.Logger,
+		}
+
+		var crdClient *k8scrdclient.CRDClient
+		crdClient, err = k8scrdclient.New(c)
+		crdBackoff := backoff.NewMaxRetries(3, 1*time.Second)
+
+		err = crdClient.EnsureCreated(ctx, v1alpha1.NewChartCRD(), crdBackoff)
```

Comment:
* We talked about this in SIG Operator and yet we try to install the CRD manually here. Why? IMO The operator should be installed and we could simply leverage `config.Release.InstallOperator`. 
* The error here is not handled.

Reply:
We have deleted creating CRD part and install chart-operator directly.